google/resource_compute_instance_test.go

```diff
@@ -1187,6 +1248,49 @@ func testAccComputeInstance_attachedDisk(disk, instance string) string {
 	}`, disk, instance)
 }
 
+func testAccComputeInstance_bootDisk(instance string) string {
+	return fmt.Sprintf(`
+	resource "google_compute_instance" "foobar" {
```

Comment:
Nit: Can you de-indent both of these configs so that the top level is against the left side of the file? That way, they are easier to copy out and modify/test interactively, or for users who browse the source to pull out pre-tested .tf configurations.